warning/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for warning 3.0.0
+// Project: https://github.com/BerkeleyTrue/warning
+// Definitions by: Chi Vinh Le <https://github.com/cvle>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const warning: (condition: any, format?: string, ...extra: any[]) => void;
```

Comment:
I can change this. Writing it this way was convenient as typescript would allow package to be imported using es6 syntax. Though to support es6 module import syntax (see below) I would have to add `declare namespace warning {}` like in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/types-2.0/object-assign/index.d.ts